<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutdoorPathsScreen</class>
 <widget class="QWidget" name="OutdoorPathsScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="backgroundLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>768</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/Outdoor Paths Background.png</pixmap>
   </property>
  </widget>
  <widget class="InvisibleButton" name="woodedButton" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>231</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text" stdset="0">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="woodedBorder">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>145</y>
     <width>231</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/Outdoor Paths Selection Border.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="deerBorder">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>145</y>
     <width>231</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/Outdoor Paths Selection Border.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="lakeBorder">
   <property name="geometry">
    <rect>
     <x>517</x>
     <y>145</y>
     <width>231</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/Outdoor Paths Selection Border.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="cityBorder">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>144</y>
     <width>231</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/Outdoor Paths Selection Border.png</pixmap>
   </property>
  </widget>
  <widget class="InvisibleButton" name="deerButton" native="true">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>150</y>
     <width>231</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text" stdset="0">
    <string/>
   </property>
  </widget>
  <widget class="InvisibleButton" name="lakeButton" native="true">
   <property name="geometry">
    <rect>
     <x>518</x>
     <y>150</y>
     <width>231</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text" stdset="0">
    <string/>
   </property>
  </widget>
  <widget class="InvisibleButton" name="cityButton" native="true">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>150</y>
     <width>231</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text" stdset="0">
    <string/>
   </property>
  </widget>
  <widget class="InvisibleButton" name="closeButton" native="true">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>30</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text" stdset="0">
    <string/>
   </property>
  </widget>
  <widget class="InvisibleButton" name="startButton" native="true">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>640</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text" stdset="0">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InvisibleButton</class>
   <extends>QWidget</extends>
   <header>invisiblebutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
